pilot/pkg/model/sidecar_test.go

```diff
@@ -2060,56 +2076,6 @@ func TestCreateSidecarScope(t *testing.T) {
 				},
 			},
 		},
-		{
```

Comment:
This test not relevant. Kubernetes service in the same namespace should be merged. The existing changed test verifies this behaviour